scripts/myrial

```diff
@@ -143,6 +146,12 @@ def main(args):
                 raise "Options dot and --plan are incompatible"
 
             print operator_to_dot(processor.get_logical_plan())
+        elif opt.opt_dot:
+            if opt.from_repr:
```

Comment:
`from_repr` specifies the *input* not the output. That is, from_repr says take a python string containing a plan rather than a file containing a myrial program.

That is why `--plan` is not in the mutually exclusive arguments group